src/server/db/schema.ts

```diff
@@ -59,6 +59,7 @@ export type Rank = z.infer<typeof rankSchema>;
 
 export const matches = bbPgTable("match", {
   id: uuid("id").primaryKey(),
+  createdAt: timestamp("created_at"),
```

Comment:
1. Make it a date, not a timestamp
2. Make it not null
3. Provide current date as default value
4. Delete migration 0004 and its data
5. Re-generate the migration with the new schema